### PR TITLE
Add indicator icon information to UG

### PR DESCRIPTION
**Pull request by nyankoclaws**

### CODE REVIEW

**Review by larainezo — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -245,6 +245,18 @@ If your changes to the data file makes its format invalid, Dormie will discard a
...
+### Indicator icons
+#### Room Number Outdated 
+<img src="images/calendar_alert.png" alt="calendar_alert" width="24"/> 
+<br>
+Shows up next to a contact when room number might be outdated. If a room number was last updated before the annual room assignments (which occur between April 5th and 12th), the alert appears. When you see this alert, it’s a good idea to check and update the room number if necessary.
+
+#### Birthday
+<img src="images/birthday_cake.png" alt="birthday_cake" width="24"/>
+<br>
+Shows up next to a contact when the contact's birthday is today. If the birthday icon shows up, it might be a good idea to wish your friend "Happy Birthday".
```
Comment (donwong2308) [inline docs/UserGuide.md:257]: Maybe can modify to "You can wish your friends a happy birthday if you see the birthday icon!"

**Review by donwong2308 — APPROVED** [review]
LGTM

**Review by PallonCX — APPROVED** [review]
LGTM

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Pull request merged**